cargo-shuttle/src/config.rs

```diff
@@ -294,8 +294,26 @@ impl RequestContext {
     }
 
     pub fn get_local_config(
-        project_args: &ProjectArgs,
+        project_args: &mut ProjectArgs
     ) -> Result<Config<LocalConfigManager, ProjectConfig>> {
+        let root_directory_path = find_root_directory(&project_args.working_directory);
```

Comment:
I see the function (`load_local()`) that calls this function (`get_local_config()`) also uses `working_directory` to read the secrets. So maybe we should move this fix even earlier in the call stack so that the secrets are read correctly too?

I'm thinking of moving it all the way up to `load_project()` in `lib.rs`. This would mean the test will have to move to `lib.rs` and will need to change to test against `load_project()`.

Reply:
I've separated the `find_root_directory()` function and wrote a test for it.

`load_project()` now calls it instead.